orchestrator/consumers/chatbot/personality.py

```
@@ -1,47 +1,126 @@
+
+        db = SessionLocal()
+        try:
+            workspace = db.query(Workspace).get(workspace_id)
+            if workspace and workspace.settings:
+                orch = workspace.settings.get("orchestrator", {})
+                for key in _ORCHESTRATOR_DEFAULTS:
+                    if key in orch:
+                        settings[key] = orch[key]
+        finally:
+            db.close()
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)